Module-5/helpers/helpers.js

```diff
@@ -0,0 +1,43 @@
+import { users } from '../data.js';
```

Comment:
this file doesn't seem `helper` to me

if you would adhere the [layered architecture](https://martinfowler.com/architecture/), functions inside this file would be `User Service`, but not helpers

everything seems to be a service function, except `handleMalformedRequest`, which is `middleware` or auxilliary method that helps to handle error cases